index.d.ts

```diff
@@ -1,11 +1,29 @@
+import {LiteralUnion} from 'type-fest';
+
 declare namespace locatePath {
+	type OptionsType = LiteralUnion<'file' | 'directory', string>;
+
 	interface Options {
 		/**
 		Current working directory.
 
 		@default process.cwd()
 		*/
 		readonly cwd?: string;
+
+		/**
+		Type of file to match.
```

Comment:
This description is not correct. It's not just `file`.

I think we should also explain that it can be set to `undefined` to match either file or directory. However, is that really useful? Should we support that at all?

Reply:
I don't see any reason to support `undefined`, it was mostly done that way to avoid being a breaking change.

If this module will only match file or directory then I think `followSymlinks` is a bad name, `allowSymlinks` (default true) or `rejectSymlinks` (default false) would be better.

Reply:
Let’s go with `allowSymlinks`. And not support `undefined`.

Reply:
Done